src/server.ts

```diff
@@ -272,7 +272,7 @@ export class Server {
   public async _getLedgerEntries(
     ...keys: xdr.LedgerKey[]
   ): Promise<SorobanRpc.RawGetLedgerEntriesResponse> {
-    return jsonrpc.post<SorobanRpc.RawGetLedgerEntriesResponse>(
```

Comment:
that's odd, i'm confused by the diff display as this wasn't changed - https://github.com/stellar/js-soroban-client/blob/raw-methods/src/server.ts#L274

Reply:
odd! maybe it's because the branch was slightly out of date